client/scripts/models/Account.ts

```diff
@@ -103,6 +103,7 @@ abstract class Account<C extends Coin> {
       const params : any = {
         address: this.address,
         chain: this.chain.id,
+        isToken: this.chain.type === "token",
```

Comment:
So to be clear, an account balance for a token forum will be the address's balance of that particular ERC20 token? (and not ETH?)

Reply:
that's my interpretation of this (and what I believe the spec is)